boxsdk/object/folder.py

```diff
@@ -382,3 +383,45 @@ def delete(self, recursive=True, etag=None):
         :raises: :class:`BoxAPIException` if the specified etag doesn't match the latest version of the folder.
         """
         return super(Folder, self).delete({'recursive': recursive}, etag)
+
+
+    def create_web_link(self, target_url, name=None, description=None):
+        """
+        Create a WebLink with a given url.
+
+        :param url:
+            The url the web link points to.
+        :type url:
+            `unicode`
+        :param parent_id:
+            The id of the parent folder where the web link is created.
+        :type parent_id:
+            `unicode`
+        :param name:
+            The name of the web link. Defaults to the URL is not specified.
```

Comment:
"if not specified"

Though I don't think we need to say what it defaults to. We can just say that it is optional, and that the API will give it some default if it isn't specified.